### PR TITLE
make the sign up BE #70

### PR DESCRIPTION
**Pull request by amanshawar**

### CODE REVIEW

**Review by FidaaHersh — APPROVED** [review]

**Review by orjwan98 — APPROVED** [review]

File: server/controllers/signup.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+const bcrypt = require('bcryptjs');
+const Airtable = require('airtable');
+const jwt = require('jsonwebtoken');
```
Comment (Hadeel-Salamin) [inline server/controllers/signup.js]: no need for 'jsonwebtoken' here

File: client/src/components/SignUp/index.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+            name="confirmPassword"
+            placeholder="**************"
+            value={this.state.confirmPassword}
+            onChange={e => this.setState({
+              confirmPassword: e.target.value,
+            })
+            }
+          />
+          <StyledP>
+            <StyledText>Been here before?</StyledText>
```
Comment (jema28) [inline client/src/components/SignUp/index.js]: There's a space missing between before? and Login.

File: client/src/components/SignUp/index.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+            value={this.state.password}
+            onChange={e => this.setState({
+              password: e.target.value,
+            })
+            }
+          />
+          <StyledLabel> Confirm password* </StyledLabel>
+          <StyledInput
+            type="password"
+            name="confirmPassword"
```
Comment (jema28) [inline client/src/components/SignUp/index.js:76]: The placeholder fonts need to be changed to Assistant. I've noticed the same in login.

File: client/src/components/SignUp/index.js [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+          />
+          <StyledP>
+            <StyledText>Been here before?</StyledText>
+            <StyledSignUp> Login</StyledSignUp>
+          </StyledP>
+        </StyledForm>
+
+        <StyledBottom>
+          <Button />
+          <GButton title="Sign up" onClick={this.signup} />
```
Comment (jema28) [inline client/src/components/SignUp/index.js]: Sign up should be in capitals.

**Review by jema28 — CHANGES_REQUESTED** [review]
Review for FE done! nice!

File: client/src/components/SignUp/index.js (hunk 4 above)
Comment (amanshawar) [reply client/src/components/SignUp/index.js]: done

File: client/src/components/SignUp/index.js (hunk 3 above)
Comment (amanshawar) [reply client/src/components/SignUp/index.js:76]: done

File: client/src/components/SignUp/index.js (hunk 2 above)
Comment (amanshawar) [reply client/src/components/SignUp/index.js]: done

**Review by Hadeel-Salamin — APPROVED** [review]

**Review by jema28 — APPROVED** [review]
💯

**Pull request merged**